javascript/javascript1/week3/notes.js

```diff
@@ -0,0 +1,32 @@
+const notes = [];
+// function to push objects to array
+function saveNote(content, id) {
+  const note = { id: id, content: content };
```

Comment:
In ES6 you have this short hand property for object. So you can write as
``` 
const note = {
id,
content,
}
```
https://javascript.plainenglish.io/object-literals-using-object-property-shorthand-6360825c60ef